packages/web3/test/fixtures/erc20.ts

```diff
@@ -0,0 +1,238 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+export const erc20Abi = [
```

Comment:
The `erc20` and `erc721` abis and other related stuff is defined in shared fixtures. Link the root level `fixtures` directory to this package as `shared_fixtures` and use these values from there.

Reply:
Currently, they are not. They exist only in `web3-eth-contract`, should I move them to `fixtures` and create symlinks in both packages symlinks? Or I miss something?